lang/zh-cn.js

```diff
@@ -68,7 +68,7 @@ export default {
 	"components.input-time-range.startTime": "开始时间",
 	"components.interactive.instructions": "按 Enter 键进行交互，按 Esc 键退出",
 	"components.link.open-in-new-window": "在新窗口中打开。",
-	"components.list.keyboard": "Use <b>arrow keys</b> to move focus inside this list, or <b>page up/down</b> to move up or down by 5",
+	"components.list.keyboard": "使用<b>箭头键</b>在此列表中移动焦点，或使用<b>上/下翻页</b>以 5 个列表项为幅度进行上移或下移",
```

Comment:
I feel like the `<b>...</b>` tag groups should be surrounded by spaces?

Reply:
Submitted this as an error:

[TR-112](https://desire2learn.atlassian.net/servicedesk/customer/portal/38/TR-112).

It may turn out that this is a quirk of these languages, but raised it anyways.

Reply:
Unlikely it needs spaces. I think they only use them where there is a break in the Chinese character set (e.g. around punctuation and Arabic/Latin numerals).